webpack/react_app/components/TargetingHosts/__tests__/HostItem.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
+import HostItem from '../components/HostItem';
+
+const fixtures = {
+  'renders HostItem': {
+    name: 'Host1',
+    link: '/host1',
+    status: 'success',
+    actions: [],
```

Comment:
can you move it into a fixtures file, the benefit is that you can reuse those props all over,
saw that you used them in other tests